package.json

```diff
@@ -7,7 +7,7 @@
     "Remoting",
     "REST"
   ],
-  "version": "3.11.0",
+  "version": "3.11.1",
```

Comment:
please revert this change, the version number is bumped automatically when we do a release.